lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -2,59 +2,35 @@
 
 const BbPromise = require('bluebird');
 const _ = require('lodash');
+const findAndGroupStacks = require('../utils/findAndGroupStacks');
+const s3ObjectsFromStacks = require('../utils/s3ObjectsFromStacks');
 
 module.exports = {
   getObjectsToRemove() {
     // 4 old ones + the one which will be uploaded after the cleanup = 5
-    const directoriesToKeepCount = 4;
-    const serviceStage = `${this.serverless.service.service}/${this.options.stage}`;
+    const stacksToKeepCount = 4;
+    const service = this.serverless.service.service;
+    const stage = this.options.stage;
```

Comment:
@pmuens @nikgraf `this.options.stage` checks `serverless.yml` [in the validate file](https://github.com/serverless/serverless/blob/master/lib/plugins/aws/lib/validate.js#L12-L17) ... so this should be good to go.

PS: those LOC checks the default property not provider, but this is for backward compatablity, the service class moves the properties from `defaults` to `provider`. So again, this should be good to go 😊